elide-integration-tests/src/test/groovy/com/yahoo/elide/tests/PaginateIT.groovy

```diff
@@ -582,6 +582,37 @@ public class PaginateIT extends AbstractIntegrationTestInitializer {
     }
 
     @Test
+    public void testPaginateAnnotationTotalsWithFilter() {
+        def result = mapper.readTree(RestAssured.get("/entityWithoutPaginate?page[size]=2&page[totals]&filter[entityWithoutPaginate.id][le]=10").asString())
+        Assert.assertEquals(result.get("data").size(), 2)
+        JsonNode pageNode = result.get("meta").get("page")
+        Assert.assertNotNull(pageNode)
+        Assert.assertEquals(pageNode.get("totalRecords").asInt(), 10)
+        Assert.assertEquals(pageNode.get("totalPages").asInt(), 5)
+    }
+
+    @Test
+    public void testRelationshipPaginateAnnotationTotals() {
+        def result = mapper.readTree(RestAssured.get("/author/${asimovId}/books?page[size]=1&page[totals]").asString())
+        Assert.assertEquals(result.get("data").size(), 1)
+        JsonNode pageNode = result.get("meta").get("page")
+        Assert.assertNotNull(pageNode)
+        Assert.assertEquals(pageNode.get("totalRecords").asInt(), 2)
+        Assert.assertEquals(pageNode.get("totalPages").asInt(), 2)
+    }
+
+    @Test
+    public void testRelationshipPaginateAnnotationTotalsWithFilter() {
+        def result = mapper.readTree(RestAssured.get("/author/${asimovId}/books?page[size]=1&page[totals]&filter[book.title][infixi]=Foundation").asString())
```

Comment:
Since this is case insensitive `infixi` perhaps change value to `fOUNDation` to be sure it is not using regular infix.